### PR TITLE
adding home page

### PR DESCRIPTION
**Pull request by ashishnnnnn**
Made the landing page of eCommerce

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,16 @@
 import "./App.css";
-import logo from "./logo.png";
-
+import { Routes, Route } from "react-router-dom";
+import { Navbar } from "./Components";
+import { Home } from "./pages";
 function App() {
-  return <div className="App"></div>;
+  return (
+    <div className="App">
```
Comment (spraveenofficial) [inline src/App.js:7]: You can remove this Div and add Fragments.

**Review by spraveenofficial — APPROVED** [review]

**Review by mnath30 — COMMENTED** [review]
Screenshot_20220317-033651_Chrome

On mobile device, the search bar is occupying the entire width of screen and it is hard to make out the entire text box of search.
Please check this and you can apply some margin to the search box so that the borders are visible and user can make out the search area.

File: src/App.js [hunk 2]
```diff
@@ -1,8 +1,16 @@
 import "./App.css";
-import logo from "./logo.png";
-
+import { Routes, Route } from "react-router-dom";
+import { Navbar } from "./Components";
+import { Home } from "./pages";
```
Comment (tushartiwari7) [inline src/App.js:4]: The Routing and namings are done very neatly. Nice 👍

File: src/Components/Sliders/Sliders.jsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+const sources = [
+  "https://rukminim2.flixcart.com/flap/1800/1800/image/bd9c3758e25371d7.jpg?q=80",
+  "https://rukminim2.flixcart.com/flap/1800/1800/image/daed16406ceeed7a.jpg?q=80",
+  "https://rukminim2.flixcart.com/flap/1800/1800/image/a0f9cae85b48f6f1.jpg?q=80",
+  "https://rukminim2.flixcart.com/flap/1800/1800/image/51890ec1d3d50ca0.jpg?q=80",
+];
+const delay = 2500;
+export const Sliders = () => {
+  const [index, setIndex] = useState(0);
+  const timeoutRef = useRef(null);
```
Comment (tushartiwari7) [inline src/Components/Sliders/Sliders.jsx:13]: Nice! you have used useRef hook.

**Review by tushartiwari7 — APPROVED** [review]
You have used best Practices and good naming conventions too nice work!

**Comment by ashishnnnnn** [comment]
Corrected the search bar view in mobile ..

**Pull request merged**